src/Temporalio/Client/ITemporalClient.Workflow.cs

```diff
@@ -92,6 +92,84 @@ WorkflowHandle<TWorkflow> GetWorkflowHandle<TWorkflow>(
         WorkflowHandle<TWorkflow, TResult> GetWorkflowHandle<TWorkflow, TResult>(
             string id, string? runId = null, string? firstExecutionRunId = null);
 
+        /// <summary>
+        /// Start an update via a call to a WorkflowUpdate attributed method, possibly starting the
```

Comment:
"via a call to a WorkflowUpdate attributed method" means "this will end up causing an update handler method to be executed in a workflow"?

Reply:
Not exactly. What "via a call" means is that this is a lambda expression that makes a call to a method with the WorkflowUpdate attribute. So it looks like `myClient.StartUpdateWithStartWorkflow((MyWorkflow wf) => wf.MyUpdateMethodWithAttribute(myArg), ...`.